utils/rate_limiter.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
although factually these files are just being copied over, shouldn't the copyright be updated here to 2018?

Reply:
Right I am wondering too. @yurishkuro what is your opinion here?

Reply:
He wrote to me yesterday: "if you copy the file, you keep C as is... if you copy and modify, and C was 2016, you should change it to C 2016, 2018"

Reply:
update this and I'll land

Reply:
update to what? This is the correct header.

Reply:
See my response above from @yurishkuro (granted it is hearsay as he didn't write it here)? It was not modified, so keeping the same.